src/frontend/mod.rs

```diff
@@ -85,6 +87,7 @@ impl<'a, B: Backend<'a>> Frontend<'a, B> {
             })),
             CmarkEvent::SoftBreak => self.buffer.push_back(Event::SoftBreak),
             CmarkEvent::HardBreak => self.buffer.push_back(Event::HardBreak),
+            CmarkEvent::TaskListMarker(checked) => self.buffer.push_back(Event::TaskListMarker(checked)),
```

Comment:
`FootnoteReference` and `Pdf` event types each have their own struct encapsulation for their data, not sure if that would be cleaner to introduce here as well.